java/vector/src/main/codegen/templates/UnionMapWriter.java

```diff
@@ -98,6 +98,18 @@ public UnionMapWriter(MapVector vector) {
     entryWriter = struct();
   }
 
+  @Override
+  public void start() {
+    writer.setAddVectorAsNullable(false);
```

Comment:
I don't think we should override these, `startMap()` should be used instead

Reply:
right, fixed.